src/Apps/ViewingRoom/Components/ViewingRoomHeader.tsx

```diff
@@ -32,7 +59,7 @@ const ViewingRoomHeaderSmall = props => {
       }}
     >
       <ResponsiveImage
-        src="https://user-images.githubusercontent.com/236943/81243255-342dcc00-8fc4-11ea-9a2b-2ab96ab67c9b.png"
+        src={heroImageURL}
```

Comment:
Since this image can be _anything_, let's pop it through gemini's `resizer` before including here to make sure it's cached behind a cdn and sized correctly. (We should also `convert_to: "jpg"` for perf reasons).

Something like: https://github.com/artsy/reaction/blob/master/src/Apps/FeatureAKG/Components/Feature.tsx#L43-L46

Reply:
Will do 👍